packages/amplication-server/src/core/resource/resource.service.ts

```diff
@@ -109,10 +110,23 @@ export class ResourceService {
       throw new InvalidColorError(color);
     }
 
+    if (args.data.resourceType === EnumResourceType.ProjectConfiguration) {
+      throw new AmplicationError(
+        'Try to create project configuration from createResource function'
+      );
+    }
+
+    const projectId = args.data.project.connect.id;
+
+    const projectConfiguration = await this.projectConfiguration(projectId);
+
     const resource = await this.prisma.resource.create({
       data: {
         ...DEFAULT_SERVICE_DATA,
         ...args.data,
+        gitRepository: {
+          connect: { id: projectConfiguration.gitRepositoryId }
```

Comment:
This will throw an exception in case projectConfiguration is null (which should not happen logically, but still)